src/Helpers/constants.js

```diff
@@ -192,6 +192,33 @@ export const PUBLIC_DATE_OPTIONS = [
     }
 ];
 
+export const SORT_CVE_OPTIONS = [
```

Comment:
This could be renamed to something like `CUSTOM_REPORT_SORT_OPTIONS` so it's obvious that this object is only used in Custom report modal and not in CVE table for example.